docs/sphinx/source/whatsnew/v0.11.2.rst

```diff
@@ -15,7 +15,9 @@ Enhancements
 Documentation
 ~~~~~~~~~~~~~
 * Edited docstrings for :py:func:`~pvlib.pvsystem.dc_ohms_from_percent` and
-  :py:func:`~pvlib.pvsystem.dc_ohmic_losses` for clarity. (:issue:`1601`, :pull:`2229`)
+  :py:func:`~pvlib.pvsystem.dc_ohmic_loss` for clarity. (:issue:`1601`, :pull:`2229`)
```

Comment:
This change should be reverted, I think.  `..._losses` is correct.

Reply:
Ah I was not trying to change the existing entry... this incomplete entry ":py:func:`~pvlib.pvsystem.dc_ohmic_losses` for clarity. (:issue:`1601`, :pull:`2229`)" just appeared, not attached to an entry, after I updated the file....
So this line should be replacing the line in the entry with "loss"?